src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -76,24 +84,28 @@ def __init__(
         do_resize=True,
         size=800,
         max_size=1333,
+        resample=Image.BILINEAR,
         size_divisibility=32,
         do_normalize=True,
         image_mean=None,
         image_std=None,
-        ignore_index=255,
+        ignore_index=None,
+        reduce_labels=False,
         **kwargs
     ):
         super().__init__(**kwargs)
         self.do_resize = do_resize
         self.size = size
         self.max_size = max_size
+        self.resample = resample
         self.size_divisibility = size_divisibility
-        self.ignore_index = ignore_index
         self.do_normalize = do_normalize
         self.image_mean = image_mean if image_mean is not None else [0.485, 0.456, 0.406]  # ImageNet mean
         self.image_std = image_std if image_std is not None else [0.229, 0.224, 0.225]  # ImageNet std
+        self.ignore_index = ignore_index
+        self.reduce_labels = reduce_labels
 
-    def _resize(self, image, size, target=None, max_size=None):
+    def _resize_with_size_divisibility(self, image, size, target=None, max_size=None):
```

Comment:
Should the docstring be updated to account for size divisibility?

Reply:
the name `_resize_with_size_divisibility` implies `size_divisibility` is passed to it but this is not the case because `size_divisibility` is a field to this class and the `resize` method is a method for this class so it's behaviour is dictated by the class itself thus I think is more correct to stick with `resize`